speech-to-text/v1.js

```diff
@@ -837,6 +838,97 @@ SpeechToTextV1.prototype.whenCustomizationReady = function(params, callback) {
   });
 };
 
+//Check if there is a corpus that is still being processed
+function isProcessing(corporaList) {
+  var recordsBeingProcessed = corporaList.corpora.filter(function(record) {
+    return record['status'] == 'being_processed';
```

Comment:
You must use `===` in order to make the linter happy. Run `npm test` locally to make sure everything will pass.